lnrpc/lightning.proto

```diff
@@ -3711,6 +3718,22 @@ message ListPermissionsResponse {
     map<string, MacaroonPermissionList> method_permissions = 1;
 }
 
+message NodeAnnouncementUpdateRequest {
+    // Features holds the set of features the node supports.
+    map<uint32, Feature> features = 1;
+
+    // Color is the node's color in hex code format.
+    string color = 2;
+
+    // Alias is the alias or nick name of the node.
+    string alias = 3;
+
+    // Addresses is a slice of all the node's known addresses.
+    repeated NodeAddress addresses = 4;
+}
+message NodeAnnouncementUpdateResponse {
```

Comment:
Not sure if it is worth to return all the data that will be broadcasted to other nodes

Reply:
Could be useful to just send what was actually updated? So like if you try to add a new address, but it's already advertised, then you get back an empty response telling you it was effectively a noop. 

Related to this response handling, is that we likely want to return a gRPC status/error code, to indicate if a user tries to do something like a set a feature bit that we actually don't understand. On the other-hand, we could allow such behavior as it may be useful when paired with something like: https://github.com/lightningnetwork/lnd/pull/5346